docs/en/tutorials/getting-started.rst

```diff
@@ -257,9 +250,9 @@ entity definition:
         }
     }
 
```

Comment:
The current documentation suggests that both getters and setters are mutators, although getters are definitely not mutators.